great_expectations/data_context/store/checkpoint_store.py

```diff
@@ -104,7 +104,7 @@ def default_checkpoints_exist(directory_path: str) -> bool:
         return os.path.isdir(checkpoints_directory_path)
 
     def list_checkpoints(
-        self, ge_cloud_mode: bool
+        self, ge_cloud_mode: bool = False
```

Comment:
We will eventually want to refactor out `ge_cloud_mode` - for now, I've opted to add a default value (as 99% of usecases will not leverage this flag).